db/migrate/20241029090758_add_role_index_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddRoleIndexToUsers < ActiveRecord::Migration[7.2]
```

Comment:
why we are applying indexing on role column? as our role column has a very limited number of distinct values.

Reply:
If the role column has a very limited number of distinct values (In our case, it is only 3), the performance gain from indexing may not be significant. But in such cases, the overhead of maintaining the index (for insertions, updates, and deletions) might outweigh the benefits.

Reply:
Got it, bhai. Will Remove this migration.
Thank you!